ops/remove-proofs-from-pcbs.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+
+require "json"
+require "find"
+
+# Recursive method to remove all "proofs" properties from a JSON structure
+def remove_proofs(obj)
+  if obj.is_a?(Hash)
+    obj.delete("proofs")
+    obj.each { |key, value| remove_proofs(value) }
+  elsif obj.is_a?(Array)
+    obj.each { |item| remove_proofs(item) }
+  end
+end
```

Comment:
Do you know if this is slower than shelling out to `jq`? We already use `jq` for other tests.

Reply:
I can't tell. It seemed very fast for me. Just a few seconds for 5000 blocks